molecule/packages-static/host_vars/centos-7

```diff
@@ -0,0 +1,2 @@
+pulp_configure_firewall: none
+pulp_python_interpreter: '/opt/rh/rh-python38/root/usr/bin/python3.8'
```

Comment:
Perhaps put a comment for pulp_python_interpreter , pulp_lib_path, pulp_smash_installed etc saying "we set this for molecule verify, not for the installer itself"?

Reply:
Done!